client/src/pages/Feed.js

```diff
@@ -47,6 +47,16 @@ import {
 } from "hooks/actions/feedActions";
 import { LOGIN } from "templates/RouteWithSubRoutes";
 
+export const isAuthorOrg = (organizations, author) => {
+  let isValid = false
+  organizations.map(org => {
```

Comment:
You can do it this way:
`const isValid = organizations.some((organization) => organization.name === author.name);`
Easier and faster.